controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentJobExecutor.java

```diff
@@ -42,6 +42,7 @@ public DeploymentJobExecutor(Controller controller, Duration triggeringInterval,
     protected void maintain() {
         controller().applications().deploymentTrigger().deploymentQueue().takeJobsToRun()
                 .forEach(buildJob -> executor.execute(() -> {
+                    log.log(Level.INFO, "Attempting to trigger " + buildJob + " in Screwdriver.");
```

Comment:
I don't see what is being logged here. The BuildJob has no toString so probably not much...
I think we should include: tenant, application, screwdriverId and job (or zone/region)

Reply:
There is a `toString`, but there is a second `BuildJob` class now, which you probably looked at.

Reply:
I'm rewriting this in any case now, so will print more information. Currently, only project ID and job type are available, and those are included in the `toString`./

Reply:
OK, but this only includes project and name